index.html

```diff
@@ -25,7 +26,174 @@ <h1>If These Lands Could Talk</h1>
 
 		<!-- Footer -->
 
+		<footer>
+			<div class="wrapper">
+				<div class="footer-main">
+					<div class="footer-logo-btn">
+						<!-- logo -->
+						<a href="./index.html">
+							<img
+								src="./assets/icons/global/itlct-logo.svg"
+								alt="If These Lands Could Talk Logo"
+							/>
+						</a>
+
+						<!-- Button for modal -->
+						<button class="btn-la">Land Acknowledgement</button>
+					</div>
+					<!-- Links list -->
+					<!-- need to replace with the real page links -->
+					<ul>
+						<li>
+							<a href="/index.html">Home</a>
+						</li>
+						<li>
+							<!-- Need to double check once Our Impact team is started -->
+							<a href="/ourimpact.html">Our Impact</a>
```

Comment:
This should probably be `/impact.html` instead of `/our-impact.html` to follow the pattern of the other links.